components/datetime/src/provider/calendar/symbols.rs

```diff
@@ -117,20 +123,76 @@ macro_rules! symbols {
 
 symbols!(
     months,
-    pub struct SymbolsV1<'data>(
-        #[cfg_attr(
-            feature = "serde",
-            serde(
-                borrow,
-                deserialize_with = "icu_provider::serde::borrow_de_utils::array_of_cow"
-            )
-        )]
-        pub [Cow<'data, str>; 12],
-    );
+    pub enum SymbolsV1<'data> {
```

Comment:
Thought: Do we want to make this an untagged enum? In terms of JSON data, it's easy to tell if this is the list version or the map version.

Reply:
We can't, that only works with self-describing formats (e.g. JSON), and will break postcard. (I tried just to be sure too). `untagged` is not a part of _serde_'s data model, it's a part of `serde_derive`'s data model, and as such serde doesn't do anything special there.